### PR TITLE
Update the spanish translation and increase some parts without translate

### PR DESCRIPTION
**Pull request by mitogh**

### CODE REVIEW

File: readme.es.md [hunk 1]
```diff
@@ -3,114 +3,151 @@
...
 
 ## Tabla de contenidos
 
-- [Entendiendo node](#entendiendo-node)
-- [Módulos en el core](#mdulos-en-el-core)
-- [Callbacks](#callbacks)
-- [Events](#events) (not written yet)
-- [Streams](#streams) (not written yet)
-- [Modules and NPM](#modules) (not written yet)
+- [Aprende node de manera interactiva](#learn-node-interactively)
```
Comment (zeke) [inline readme.es.md]: These links will need to be updated to work properly. `#learn-node-interactively` should be `#aprende-node-de-manera-interactiva`, and so on. You can refer to the DOM IDs generated by GitHub here: https://github.com/maxogden/art-of-node/blob/38d9d56a7daaa15bd4b086c17ab7db9bdb91684d/readme.es.md

**Comment by zeke** [comment]
Thanks, @mitogh! I left a comment inline about some links that need to be updated. Once that is fixed, I will happily merge this.

:link: :link: :link:

**Comment by mitogh** [comment]
Sounds pretty good, will those links thank you.

**Comment by mitogh** [comment]
Would you mind closing this one @zeke I will send another pull because I deleted the old fork I use to have.

Thank you!

**Pull request closed without merging**